test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -119,17 +121,16 @@ def test_ec2_tensorflow_inference_gpu_tensorrt(
             inference_string=f"""'{{"instances": [[{",".join([str([1]*28)]*28)}]]}}'""",
         )
         assert test_results, "TensorRt test failed!"
-    except:
-        tensorrt_test_failed = True
+    except Exception:
         remote_out = ec2_connection.run(
             f"docker logs {serving_container_name}", warn=True, hide=True
         )
         LOGGER.info(
             f"--- TF container logs ---\n--- STDOUT ---\n{remote_out.stdout}\n--- STDERR ---\n{remote_out.stderr}"
         )
+        raise
     finally:
         ec2_connection.run(f"docker rm -f {serving_container_name}", warn=True, hide=True)
-    assert not tensorrt_test_failed, "TensorRt tests have failed - please take a look at the logs."
```

Comment:
Removed this because it obfuscates the actual exception and traceback in the test logs. Instead, we can just raise the encountered exception after performing the required error handling.